compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/CanvasLayersComposeScene.skiko.kt

```diff
@@ -150,6 +151,14 @@ private class CanvasLayersComposeSceneImpl(
         focusOwner = { focusedOwner.focusOwner }
     )
 
+    override fun activeDragAndDropManager(): PlatformDragAndDropManager {
+        layers.fastForEachReversed {
+            if (it.focusable)
+                return it.owner.dragAndDropManager
+        }
+        return mainOwner.dragAndDropManager
```

Comment:
```suggestion
        return focusedOwner.dragAndDropManager
```

Reply:
Are you sure? There's an additional `isInteractive` check in `requestFocus` before `focusedLayer` is assigned a newly added layer, and I'm not sure what it does exactly.

Reply:
You need currently focused value. This property is exactly what you need

Reply:
done